src/modules/deposit/service.ts

```diff
@@ -86,16 +89,11 @@ export class DepositService {
  */
 export function formatDeposit(deposit: Deposit) {
   return {
-    depositId: deposit.depositId,
     depositTime: Math.round(DateTime.fromISO(deposit.depositDate.toISOString()).toSeconds()),
-    status: deposit.status,
-    filled: deposit.filled,
     sourceChainId: deposit.sourceChainId,
     destinationChainId: deposit.destinationChainId,
     assetAddr: deposit.tokenAddr,
-    amount: deposit.amount,
-    depositTxHash: deposit.depositTxHash,
     fillTxs: deposit.fillTxs.map(({ hash }) => hash),
-    depositRelayerFeePct: deposit.depositRelayerFeePct,
+    ...deposit,
```

Comment:
The reason for which I didn't use the spread operator is the fact that I didn't want to expose all the `deposit` table fields to the FE, only the needed ones. I don't have a strong opinion on this, but I prefer to not use the spread operator

Reply:
Ok removed the spread operator for now. If we need additional fields in the FE then we can later evaluate if spreading would make sense.